rts/Game/Game.cpp

```diff
@@ -1277,7 +1290,7 @@ bool CGame::UpdateUnsynced(const spring_time currentTime)
 	globalRendering->lastFrameStart = currentTime;
 	// Update the interpolation coefficient (globalRendering->timeOffset)
 	if (!gs->paused && !IsSimLagging() && !gs->PreSimFrame() && !videoCapturing->AllowRecord()) {
-		globalRendering->weightedSpeedFactor = 0.001f * gu->simFPS; 
```

Comment:
random whitespace change? belongs in a separate PR imo

Reply:
My guess is that this is from an IDE doing it automatically, some IDEs remove spaces at the end of lines automatically.

Reply:
that ^
I don't know how to edit past commits, is making a new commit that reverts this and more random whitespace changes going to mess with things?

Reply:
At this point, please make it a new commit. I've already cleaned up some stuff partially when cleaning up branch and doing it more will be quite time consuming.

Reply:
`git rebase` can be used to edit past commits, but `git rebase`should always be used with great care, since it can be easy to mess up and destroy changes or mess up the change history.